### ISSUE TITLE
[Error]: Collection "@nrwl/jest" cannot be resolved

### ISSUE THREAD

**Issue by andyjessop** [nrwl/nx#4353]
## Current Behavior
```bash
nx create-nx-workspace mymonorepo
nx generate library state
```

Generates error:
```bash
CollectionCannotBeResolvedException [Error]: Collection "@nrwl/jest" cannot be resolved.
    at NodeModulesEngineHost.resolve (/Users/andy/dev/cruxnx/crux/node_modules/@angular-devkit/schematics/tools/node-module-engine-host.js:75:19)
    at NodeModulesEngineHost._resolveCollectionPath (/Users/andy/dev/cruxnx/crux/node_modules/@angular-devkit/schematics/tools/node-module-engine-host.js:80:37)
    at NodeModulesEngineHost.createCollectionDescription (/Users/andy/dev/cruxnx/crux/node_modules/@angular-devkit/schematics/tools/file-system-engine-host-base.js:110:27)
    at SchematicEngine._createCollectionDescription (/Users/andy/dev/cruxnx/crux/node_modules/@angular-devkit/schematics/src/engine/engine.js:148:40)
    at SchematicEngine.createCollection (/Users/andy/dev/cruxnx/crux/node_modules/@angular-devkit/schematics/src/engine/engine.js:141:43)
    at /Users/andy/dev/cruxnx/crux/node_modules/@angular-devkit/schematics/src/rules/schematic.js:24:43
    at MergeMapSubscriber.project (/Users/andy/dev/cruxnx/crux/node_modules/@angular-devkit/schematics/src/rules/call.js:75:24)
    at MergeMapSubscriber._tryNext (/Users/andy/dev/cruxnx/crux/node_modules/rxjs/internal/operators/mergeMap.js:67:27)
    at MergeMapSubscriber._next (/Users/andy/dev/cruxnx/crux/node_modules/rxjs/internal/operators/mergeMap.js:57:18)
    at MergeMapSubscriber.Subscriber.next (/Users/andy/dev/cruxnx/crux/node_modules/rxjs/internal/Subscriber.js:66:18)
Collection "@nrwl/jest" cannot be resolved.
```

## Expected Behavior
I expected this to generate a new library called "state"

## Steps to Reproduce
As above

### Environment
```bash
  Node : 13.11.0
  OS   : darwin x64
  npm  : 6.13.7

  nx : Not Found
  @nrwl/angular : Not Found
  @nrwl/cli : 11.0.15
  @nrwl/cypress : Not Found
  @nrwl/devkit : 11.0.15
  @nrwl/eslint-plugin-nx : Not Found
  @nrwl/express : Not Found
  @nrwl/jest : Not Found
  @nrwl/linter : Not Found
  @nrwl/nest : Not Found
  @nrwl/next : Not Found
  @nrwl/node : Not Found
  @nrwl/react : Not Found
  @nrwl/schematics : Not Found
  @nrwl/tao : 11.0.15
  @nrwl/web : Not Found
  @nrwl/workspace : 11.0.15
  typescript : 4.0.5
```

**Comment by vsavkin** [comment]
@FrozenPandaz we don't have the Jest dependency when generating a workspace lib in an empty workspace.

**Comment by FrozenPandaz** [comment]
This is now fixed with #4454
